synapse/tests/test_utils_stormcov.py

```diff
@@ -96,7 +96,7 @@ async def pullone(genr):
                 async def pullgenr():
                     frame = inspect.currentframe()
                     assert 'spin.storm' in stormtracer.dynamic_source_filename(None, frame)
-                    assert (2, 2) == stormtracer.line_number_range(frame)
+                    assert (3, 3) == stormtracer.line_number_range(frame)
```

Comment:
should be able to remove `StormReporter.translate_lines` now as well since we don't need to handle empty lines